build-image/openhabian.conf

```diff
@@ -43,6 +43,10 @@ ipv6=enable
 repositoryurl=https://github.com/openhab/openhabian.git
 clonebranch=openHAB3
 
+# initial openHAB configuration to import (URL or filename)
+# the file needs to be a zip archive created from openhab-cli backup
+initialconfig=/boot/initial.zip
+
```

Comment:
Don't we want this to be commented out with something like `/boot/foo.zip` as a default for consistency?